content/handbook/support/workflows/dedicated_logs.md

```
@@ -0,0 +1,348 @@
+category: GitLab Dedicated
+description: "GitLab Dedicated サポート - ログの取り扱い"
+upstream_path: /handbook/support/workflows/dedicated_logs/
+upstream_sha: 47fdb6582389288bed0f04a23aa5d972c3ce1ff5
+translated_at: "2026-05-08T21:00:00Z"
+translator: claude
+stale: false
+---
+
+## ログの取り扱い
+
+サポートは、私たちの [OpenSearch](https://opensearch.org/) インフラストラクチャを通じて GitLab Dedicated テナントのログにアクセスできます。開始するには [ログへのアクセス](#accessing-logs) を参照してください。[OpenSearch](https://opensearch.org/) は [Kibana](/handbook/support/workflows/kibana/) のように使えますが、違いについては [ログの検索](#searching-logs) を参照してください。
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)